### DISCUSSION TITLE
Jan 30

### DISCUSSION THREAD

**Discussion by hanseulhee** [GymHubCommunity/GymHub-FE#73] [DailyMeeting]
### ⚽️ 회의 내용

- [x] 개발 내용 공유

마커 클러스터러

### 🏀 오늘 할 일

슬희 - 구글 oAuth 프로필 이미지. 카카오 oAuth
수 - 날짜 입력 폼, 회원가입 validation
수영 - 지도 내위치 기능
윤혁 - 태그 입력창

### 🎱 TMI

슬희 - 건강하게 살기 .. 6시 비행기 ~~ ✔️
수 - 더 더 더 건강하게 살기 .. pt 깜빡 .. !! pt 쌤 진짜 이상 ...
수영 - 아침 운동 갔오용 카페 왔음 ~~ 부지런tv ...
윤혁 - 커팅 .. 컷팅 .. 커팅 여름 전에 ~~ 솔로지옥 ㄱ ㄱ

**Comment by miniposi** [comment]
어랏 이미 만들어두셨군요~~
